students/Hebdowska_Grazyna/lesson_01_basic/Sum_of_digits.py

```diff
@@ -0,0 +1,8 @@
+number = input("Give numer like xxx:")
+nu = str(number)
+setne = nu[0]
```

Comment:
Please try to avoid mixing Polish and English. Use full English word for variable name
e.g
number, 
number_str, 
sum instead of sum_